code/online_challenges/src/hackerrank/Lonely Integer/Lonely_Integer.cpp

```diff
@@ -0,0 +1,76 @@
+#include <bits/stdc++.h>
+
+using namespace std;
+
+vector<string> split_string(string);
```

Comment:
All functions should be lower camel case.

Reply:
@arnavb This is actually the inbuilt hackerrank template so we can't make any changes.
It was a functional problem

Reply:
Ok.